src/app/cards/[card_id]/edit/_components/EditCard.tsx

```diff
@@ -0,0 +1,303 @@
+'use client';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { Skeleton } from '@mantine/core';
+import { useParams, useRouter } from 'next/navigation';
+import { useCallback } from 'react';
+import { SelectQRCodeModal } from './SelectQRCodeModal';
+import { UpdateCardSchemaType, updateCardSchema } from './schema';
+import { Dto_BusinessCardResponse } from '@/api/@types';
+import { Button } from '@/shared/components/common/Button';
+import { Container } from '@/shared/components/common/Container';
+import { TextInput, Textarea } from '@/shared/components/common/Input';
+import { Center, Group, Paper, Stack } from '@/shared/components/common/Layout';
+import { Loader } from '@/shared/components/common/Loader';
+import { QRCode } from '@/shared/components/common/QRCode';
+import { Text } from '@/shared/components/common/Text';
+import { BusinessCard, getQRCodeUrl } from '@/shared/components/features';
+import { BusinessCardDesignModal } from '@/shared/components/features/BusinessCard/BusinessCardDesignModal';
+import { ROUTES } from '@/shared/constants';
+import { useGetArAssets } from '@/shared/hooks/restapi/v1/ArAssets';
+import {
+  useGetBusinessCard,
+  useUpdateBusinessCard,
+} from '@/shared/hooks/restapi/v1/BusinessCard';
+import { useForm } from '@/shared/hooks/useForm';
+import { useNotifications } from '@/shared/hooks/useNotifications';
+import { useDisclosure } from '@/shared/lib/mantine';
+import { useLoading } from '@/shared/providers/loading';
+import { getAddressFromZipcode } from '@/shared/utils/address';
+
+export const EditCard = () => {
```

Comment:
```suggestion
export const EditCard = ({id}: {id: string}) => {
```
propsでpage.tsxから値を受け取るように